pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -28,14 +28,14 @@
 with python3Packages;
 buildPythonApplication rec {
   pname = "kitty";
-  version = "0.26.2";
+  version = "0.26.5";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "kovidgoyal";
     repo = "kitty";
-    rev = "v${version}";
-    sha256 = "sha256-IqXRkKzOfqWolH/534nmM2R/69olhFOk6wbbF4ifRd0=";
+    rev = "refs/tags/v${version}";
```

Comment:
Why this change? :thinking: @r-ryantm @ryantm

Reply:
https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8